commands/src/api/mod.rs

```diff
@@ -12,16 +12,35 @@ pub enum Error {
     DeserializeCommand,
 }
 
-/// API request packet type
+/// API message type
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum Message {
+    /// Request
+    Request(Request),
+    /// Notification
+    Notification(Notification),
+}
+
+/// Request type
+/// Expect response from remote end
```

Comment:
Since this is markdown, these two lines will get merged together. To make these two paragraphs, there has to be an extra newline. Also, I think it should be `Expected`, right?

```suggestion
/// Request type
///
/// Expected response from remote end
```